gradio/templates.py

```diff
@@ -97,7 +106,9 @@ class Pil(components.Image):
     is_template = True
 
     def __init__(self, **kwargs):
```

Comment:
Wouldn't it be better if we specified the kwargs the class supports as opposed to accepting arbitrary kwargs?

Right now you could do `gr.Pil(type="numpy")` which is confusing/defeats the purpose of the template. Also means devs won't get hints from their editors and makes it easy to make a typo and only be aware at runtime.

Reply:
Ah yeah you're right. I think it's okay to override the template params but you're right it would be good to specify them explicitly